platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/LdapClaimsHandler.java

```diff
@@ -112,8 +116,34 @@ public ProcessedClaimCollection retrieveClaimValues(ClaimCollection claims,
 
             connection = connectionFactory.getConnection();
             if (connection != null) {
-                BindResult bindResult = connection.bind(bindUserDN,
-                        bindUserCredentials.toCharArray());
+                BindRequest request;
```

Comment:
regardless of what happens i vote for getting rid of case sensitivity since it makes no impact in this case. SIMPLE, Simple, or SiMpLe, it aint gonna matter and is confusing to configurators.